ignite-3/src/jepsen/ignite3.clj

```diff
@@ -137,11 +137,12 @@
       (info node "Installing" (:flavour test) version)
       (c/su
         (c/exec :mkdir :-p server-dir)
-        (cu/install-archive! (:url test) (db-dir test))
-        (when (= 0 (.indexOf (:nodes test) node))
-          (cu/install-archive! (:cli-url test) (cli-dir test)))
-        (configure-server! test node)
-        (start! test node)))
+        (c/exec :chown :-R (:username (:ssh test)) server-dir))
```

Comment:
How a user could chown folder if there is a file owned by someone else?

Reply:
It's included into `(c/su ..)` form.

<img width="447" alt="selection" src="https://github.com/user-attachments/assets/e6f1ef72-f5fe-4ae6-bbdc-79170c57057f">

All other forms below it are executed without `su`, from the current user.